### PR TITLE
add StringField, fix todos on MarkdownField

### PR DESCRIPTION
**Pull request by cdcarter**
57228863339__50708031-67B4-4C8D-AAD8-5220CB92AB01-1

I meant to do those TODOs for MarkdownField a while ago. 
- Instances of MarkdownField can override `allowed_attrs` and `allowed_tags` as kwargs in the model definition. Previously, a custom allow list would require a subclass of MarkdownField. (Since there are no known subclasses of this field yet, I didn't sweat the fact that I renamed the properties.)
- You can also now pass `is_safe=False` as a kwarg to the MarkdownField constructor to prevent the string from being `mark_safe`'d.

And:
- Adds `StringField` to the fields package, which has no required max_length field but renders a single line text widget.

### CODE REVIEW

File: sfdo_template_helpers/fields/markdown.py [hunk 1]
```diff
@@ -70,19 +70,34 @@ class MarkdownFieldMixin:
...
         name, path, args, kwargs = super().deconstruct()
         # Only include kwarg if it's not the default
-        if self.property_suffix != DEFAULT_SUFFIX:
+        if self.property_suffix != MarkdownField.property_suffix:
             kwargs["property_suffix"] = self.property_suffix
+        if not self.is_safe:
+            kwargs["is_safe"] = self.is_safe
+        if self.allowed_tags != MarkdownField.allowed_tags:
+            kwargs["allowed_tags"] = self.allowed_tags
+        if self.allowed_attrs != MarkdownField.allowed_attrs:
```
Comment (cdcarter) [inline sfdo_template_helpers/fields/markdown.py]: would love some way of making this pattern (and its pair in `__init__`) quicker but meh

File: sfdo_template_helpers/fields/markdown.py [hunk 2]
```diff
@@ -29,11 +28,12 @@ def __get__(self, instance, owner):
         raw_md = instance.__dict__[self.field.name]
         if raw_md is None:
             return ""
-        return mark_safe(
+        safer = mark_safe if self.field.is_safe else supplier
```
Comment (cdcarter) [inline sfdo_template_helpers/fields/markdown.py]: am I just writing java in python here?

**Comment by jlantz** [comment]
@cdcarter thanks for the contribution :)

**Comment by cdcarter** [comment]
Very annoyingly, the django admin still renders StringField with a textarea widget (tho it has the correct  FormField class, so a ModelForm works just as expected!) The logic is kinda bananas, if django-admin has a widget specified for *any class in the MRO* for your model class, it will use that override. Trying to figure out a way to solve that without requiring a custom ModelAdmin....

**Comment by davisagli** [comment]
closing in favor of #24 where I made a couple adjustments

**Pull request closed without merging**